README.md

```diff
@@ -496,6 +496,21 @@ deleteAll() async {
 
 ```
 
+## Environment Variables
+
+Environment variables are variables like those we create in our applications.
```

Comment:
@miguelmartins17 rather than explaining what environment variables are (_incompletely_),
we should just point people to: https://github.com/dwyl/learn-environment-variables#what